test/e2e/gardener/shoot/create_update_delete.go

```diff
@@ -50,7 +50,7 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 
 	It("Create, Update, Delete", Label("simple"), func() {
 		By("Create Shoot")
-		ctx, cancel := context.WithTimeout(parentCtx, 20*time.Minute)
+		ctx, cancel := context.WithTimeout(parentCtx, 30*time.Minute)
```

Comment:
Why was this necessary?

Reply:
In the automated tests, it seems the shoot cluster creation started even though another rollout of istio was ongoing. Therefore, the seed becomes not ready leading to the gardenlet waiting for it to become ready. The rollout takes up to 15 minutes, which would not give the actual shoot cluster creation enough time to succeed. Therefore, I increased the timeout.
I will try to understand why another istio rollout is happening.

Reply:
The first revision of the istio ingress gateway deployment does not have `topologySpreadConstraints` while the second revision have them.
Do you have any idea why this should happen? I would have hoped that the istio ingress gateway would be directly mutated by the high-availability webhook.

Reply:
I can only explain it with a stale cache in `gardener-resource-manager` not observing the HA labels/annotations on the namespace while the istio deployment already got created. 🤷🏻